docs/repositories-configure/removing-your-repository.md

```diff
@@ -16,4 +16,4 @@ To delete your repository from Codacy:
     ![Removing your repository](images/repository-remove.png)
 
     !!! note
-        For added security, after you remove the repository from Codacy you can delete the webhooks and SSH keys related to this Codacy repository from your Git provider to prevent their reuse.
+        For added security, after you remove the repository from Codacy you can delete the webhooks and SSH keys (GitLab and Bitbucket only) related to this Codacy repository from your Git provider to prevent their reuse.
```

Comment:
The recommendation to delete webhooks applies to all three providers, the limitation applies only to SSH keys. Maybe it's not clear enough?

Adding an alternative below, but I think the sentence got more complex... 🤔 
Thought? 

```suggestion
        For added security, after you remove the repository from Codacy you can delete from your Git provider the resources related to this Codacy repository to prevent their reuse:
        -   Webwooks
        -   SSH keys (GitLab and Bitbucket only)
```

Reply:
True, the sentence is a bit more complex, but it's much clearer that webhooks apply to all three. Looks better to me like this, yes 👍

Reply:
Something went wrong with this commit, I think I committed the wrong suggestion. Fixing this manually.